test/CMakeLists.txt

```diff
@@ -1143,17 +1143,17 @@ ecbuild_add_test( TARGET  test_${PROJECT_NAME}_tropomi_no2_tropo
                           tropomi_no2_tropo.nc ${IODA_CONV_COMP_TOL})
 
 # disable this test 
-#ecbuild_add_test( TARGET  test_${PROJECT_NAME}_mopitt_co
-#                  TYPE    SCRIPT
-#                  ENVIRONMENT "PYTHONPATH=${IODACONV_PYTHONPATH}"
-#                  COMMAND bash
-#                  ARGS    ${CMAKE_BINARY_DIR}/bin/iodaconv_comp.sh
-#                          netcdf
-#                          "${Python3_EXECUTABLE} ${CMAKE_BINARY_DIR}/bin/mopitt_co_nc2ioda.py
-#                          -i testinput/mopitt_co.he5
-#                          -o testrun/mopitt_co.nc
-#                          -r 2021092903 2021092921"
-#                  mopitt_co.nc ${IODA_CONV_COMP_TOL_ZERO})
+ecbuild_add_test( TARGET  test_${PROJECT_NAME}_mopitt_co
+                  TYPE    SCRIPT
+                  ENVIRONMENT "PYTHONPATH=${IODACONV_PYTHONPATH}"
+                  COMMAND bash
+                  ARGS    ${CMAKE_BINARY_DIR}/bin/iodaconv_comp.sh
+                          netcdf
+                          "${Python3_EXECUTABLE} ${CMAKE_BINARY_DIR}/bin/mopitt_co_nc2ioda.py
+                          -i testinput/mopitt_co.h5
```

Comment:
we need to add it back as `.he5`

do whatever we need to do as these are correct file extensions.. think it involves re-writing the git lfs log